src/main/resources/ca/corefacility/bioinformatics/irida/export/ncbi.xml

```diff
@@ -18,9 +18,16 @@
 
 	<Action th:each="sample : ${submission.bioSampleFiles}">
 		<AddFiles target_db="SRA" th:each="file : ${sample.files}">
-			<File th:attr="file_path=${file.getId() + '.fastq'}">
-				<DataType>generic-data</DataType>
-			</File>
+			<div th:if="${file.getFiles().toString().endsWith(&quot;.gz&quot;)}">
```

Comment:
Rather than a `<div>` please use a `<th:block>` instead.  The `th:block` won't show up in the resulting xml like the `div` will.  

https://www.thymeleaf.org/doc/tutorials/2.1/usingthymeleaf.html#synthetic-thblock-tag